pythonforandroid/recipes/sdl2/__init__.py

```diff
@@ -24,12 +24,12 @@ def get_recipe_env(self, arch=None):
         env['PYTHON_INCLUDE_ROOT'] = self.ctx.python_recipe.include_root(arch.arch)
         env['PYTHON_LINK_ROOT'] = self.ctx.python_recipe.link_root(arch.arch)
 
-        if 'python2' in self.ctx.recipe_build_order:
-            env['EXTRA_LDLIBS'] = ' -lpython2.7'
-
-        if 'python3' in self.ctx.recipe_build_order:
-            env['EXTRA_LDLIBS'] = ' -lpython{}m'.format(
+        if 'python2' in self.ctx.recipe_build_order or \
+                'python3' in self.ctx.recipe_build_order:
```

Comment:
I know people would probably not like it, but I have to give it a try :joy: 
```python
if set('python2', 'python3') & set(self.ctx.recipe_build_order):
```

Reply:
If you want to write it neatly, `if self.ctx.python_recipe.name in ('python2', 'python3')` seems nicer. It seems a minor point though.